### ISSUE TITLE
Item is weirdly offset from the grid

### ISSUE THREAD

**Issue by hp5c** [Frexsim/grid-pack#7]
![image](https://github.com/Frexsim/grid-pack/assets/87440868/82c60fd4-8d14-4cca-b6ad-1453c25467ef)
Was trying out gridpack until i decided to change up the script a little to be compatible with frames and it made the items follow some weird offset.

**Comment by Frexsim** [comment]
Could you please take a screenshot of your hierarchy/explorer and post it here? My best guess right now is that the UIGridLayout's horizontal and vertical alignment is centered.

If you want the grid to scale with the background then set the grid's SlotAspectRatio to nil.

**Comment by hp5c** [comment]
![image](https://github.com/Frexsim/grid-pack/assets/87440868/d0c13ebb-7eca-4886-8ed7-70557319b6e3)
changed alot of things regarding the gui but still facing the same problem.

_my grid if it matters_
```
local pocketsGrid = GridPack.createGrid({
	
	Parent = pocketsInventory,
	Visible = true,

	Assets = {
		Slot = game:GetService("ReplicatedStorage"):WaitForChild("Assets"):WaitForChild("GUI"):WaitForChild("gridCanvasGroup"),
	},

	GridSize = Vector2.new(10, 2),
	SlotAspectRatio = nil,

	AnchorPoint = Vector2.new(0,0), 
	Position = UDim2.new(0,0),
	Size = UDim2.fromScale(1,1),
})
```

**Comment by hp5c** [comment]
[gridinventory.zip](https://github.com/Frexsim/grid-pack/files/15181242/gridinventory.zip)
i dumbed down the hierarchy a tad bit so its less clutter for you

**Comment by Frexsim** [comment]
I think I've found the problem! But I don't have time to fix it right now, so expect a fix tomorrow!

**Comment by hp5c** [comment]
thanks!

**Comment by Frexsim** [comment]
Now fixed in v0.2.2-alpha!
